Rebus.MongoDb2/MongoDbTimeoutStorage.cs

```diff
@@ -20,19 +20,30 @@ public class MongoDbTimeoutStorage : IStoreTimeouts
         const string SagaIdProperty = "saga_id";
         const string DataProperty = "data";
         const string IdProperty = "_id";
+        const string DueLockProperty = "due_lock";
+        const double DefaultLockTimeOffsetInSeconds = 1.0;
         readonly IMongoCollection<BsonDocument> collection;
+        readonly TimeSpan _lockTimeoutsOffset;
 
         /// <summary>
         /// Constructs the timeout storage, connecting to the Mongo database pointed to by the given connection string,
-        /// storing the timeouts in the given collection
+        /// storing the timeouts in the given collection with specified lockTimeoutsOffset
         /// </summary>
-        public MongoDbTimeoutStorage(string connectionString, string collectionName)
+        public MongoDbTimeoutStorage(string connectionString, string collectionName, TimeSpan? lockTimeoutsOffset = null)
         {
+            if (lockTimeoutsOffset == null)
+                lockTimeoutsOffset = TimeSpan.FromSeconds(DefaultLockTimeOffsetInSeconds);
+
+            if (lockTimeoutsOffset <= TimeSpan.FromSeconds(0.3))
```

Comment:
Can we avoid using 0.3 as constant? Even more, can we use reference a value from where this limit is taken from?

Reply:
Constant declared.